pkg/asset/installconfig/ibmcloud/validation_test.go

```diff
@@ -449,7 +449,7 @@ func TestValidatePreExitingPublicDNS(t *testing.T) {
 
 	dnsRecordName := fmt.Sprintf("api.%s.%s", validClusterName, validBaseDomain)
 
-	metadata := ibmcloud.NewMetadata(validBaseDomain, "us-south")
+	metadata := ibmcloud.NewMetadata(validBaseDomain, "us-south", nil, nil)
```

Comment:
is any additional unit test coverage possible with the addition of these new parameters?

Reply:
Let me take a look quick

Reply:
I think we have some testing coverage for the subnets (these added parameters) as part of [TestValidate](https://github.com/openshift/installer/blob/fdd3902490cc1e0980e9e15b120ffc4cb2123fdc/pkg/asset/installconfig/ibmcloud/validation_test.go#L217-L328).

And since neither of those parameters (control plane subnets and compute subnets) are used in the `ValidatePreExitingPublicDNS` code, I think it's okay to just default them to `nil` and basically ignore them.

I will see about adding a `metadata_test.go` suite in a future PR however, so we have more fine grained testing of the `metadata.go` functions. Perhaps something I can get done next week or so. If that's cool.